images/gray.svg

```diff
@@ -39,5 +37,5 @@
    height="45"
    width="45"
    id="rect816"
-   style="fill:#c0c0c0;fill-opacity:0.1;stroke:none;stroke-width:23.84326935;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
+   style="fill:#c0c0c0;fill-opacity:0.0;stroke:none;stroke-width:23.84326935;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
```

Comment:
Do we need this element at all  now that it is transparent?

Reply:
Right. I think we can entirely do without it.

Reply:
Right. We can entirely do without it.